include/swift/AST/Decl.h

```diff
@@ -5051,6 +5056,14 @@ class VarDecl : public AbstractStorageDecl {
   /// From the standpoint of access control and exportability checking, this
   /// var will behave as if it was public, even if it is internal or private.
   bool isLayoutExposedToClients() const;
+  
+  /// Is the declaration captured in an #if-config?
```

Comment:
You could remove this doc comment, since the doc comment for `DiagnoseUsage` already explains the meaning in more detail.